src/main/java/ch/njol/skript/bukkitutil/ItemUtils.java

```diff
@@ -379,4 +380,42 @@ public static boolean isGlass(Material material) {
 				return false;
 		}
 	}
+
+	/**
+	 * Applies a provided {@code Consumer} to the meta of the provided {@code ItemStack} and returns the updated {@code ItemStack} (with updated {@code ItemMeta}).
+	 *
+	 * @param itemStack the item whose meta is to be changed using the provided Consumer
+	 * @param metaChanger a consumer to update the meta of the provided ItemStack
+	 * @param <T>
+	 * @return the updated item
+	 */
+	public static <T extends ItemMeta> T changeItemMeta(@NotNull ItemStack itemStack, @NotNull Consumer<T> metaChanger) {
+		//noinspection unchecked
+		T itemMeta = (T) itemStack.getItemMeta();
+		metaChanger.accept(itemMeta);
+		return itemMeta;
+	}
+
+	/**
+	 * Updates the provided object's ({@code Slot}, {@code ItemType}, {@code ItemStack}) {@link ItemMeta} by setting it to the provided {@code ItemStack}.
+	 *
+	 * @param object the object to update
+	 * @param itemMeta the {@link ItemMeta} to change to
+	 * @see #asItemStack(Object)
+	 */
+	public static void setItemMeta(Object object, @NotNull ItemMeta itemMeta) {
+		if (object instanceof Slot slot) {
+			ItemStack itemStack = slot.getItem();
+			if (itemStack == null)
+				return;
+			itemStack.setItemMeta(itemMeta);
+			slot.setItem(itemStack);
+		} else if (object instanceof ItemType itemType) {
+			itemType.setItemMeta(itemMeta);
+		} else if (object instanceof ItemStack itemStack1) {
+			itemStack1.setItemMeta(itemMeta);
```

Comment:
```suggestion
		} else if (object instanceof ItemStack itemStack) {
			itemStack.setItemMeta(itemMeta);
```
The other `itemStack` is in a different scope, so this should be okay?